.travis.yml

```diff
@@ -23,6 +23,6 @@ before_script:
   - kind load docker-image kubernetes/pause
   - kind get kubeconfig > /tmp/admin.conf
 script:
-- mkdir -p ~/gopath/src/sigs.k8s.io/
-- mv ~/gopath/src/github.com/kubernetes-sigs/descheduler ~/gopath/src/sigs.k8s.io/.
-- make test-e2e
+  - mkdir -p ~/gopath/src/sigs.k8s.io/
+  - mv ~/gopath/src/github.com/kubernetes-sigs/descheduler ~/gopath/src/sigs.k8s.io/.
+  - make test-e2e
```

Comment:
@stevehipwell please remove the leading spaces here. This is not required in the PR. Thanks!